### DISCUSSION TITLE
License of the project

### DISCUSSION THREAD

**Discussion by ichxorya** [useocl/use#98] [Ideas]
The project is currently using the license GNU GPL v2. While it's not causing any big trouble, I think it's better to get updated with GPL v3, as v2 was released back in 1991 (and probably was used for legacy reasons).

You can read this post to learn more why we should update the license: https://www.gnu.org/licenses/rms-why-gplv3.html

For my personal opinion, I don't like GPL for its viral nature (see https://en.wikipedia.org/wiki/Copyleft#%22Viral%22_licensing). We should see if other licenses are more suitable, i.e. LGPL, MIT, ect.

**Comment by h-man2** [comment]
You are right with legacy reasons: the development of USE started around 1996.

As you already mentioned, this does not cause big trouble. But feel free to evaluate the different areas that need to be chaned: I would guess the file headers and some readme files.

On einteresting questin for me is: who can change the license? Anyone with ownerchip of a repository?

**Reply by ichxorya** [reply]
As the owner & main maintainer of this project, I guess you have the right to be some kind of "project director" or such (which means you & your team can decide what to do with it). You can have a meeting with your peers to talk about it, or even bring it up as a discussion topic on this Github Discussion section.
